pkg/reconciler/buildrun/buildrun.go

```diff
@@ -466,6 +466,27 @@ func (r *ReconcileBuildRun) Reconcile(ctx context.Context, request reconcile.Req
 				}
 			}
 
+			if buildRun.Spec.Output != nil {
```

Comment:
can u add some comment on what is happening here pls?

I assume here we are doing the override between the properties of Vulscan between the Build vs the BuildRun, where the Buildrun should override?

Reply:
I changed the code, so that both the cases are handled, if the vulnerability scan options are specified either on the build spec or the buildrun spec.

Reply:
We discussed that the contents of `failOnVulnerabilities` should not consider the embedded build spec output, but rather to use the output if the Build(if exists)